python_modules/libraries/dagster-dg/dagster_dg/cli/scaffold.py

```diff
@@ -182,6 +183,45 @@ def scaffold_defs_group(context: click.Context, help_: bool, **global_options: o
         context.exit(0)
 
 
+# ########################
+# ##### DEFS INLINE-COMPONENT
+# ########################
+
+
+@scaffold_defs_group.command(
+    name="inline-component",
+    cls=ScaffoldDefsSubCommand,
+)
+@click.argument("path", type=str)
+@click.argument("typename", type=str)
```

Comment:
I think this should be a flag so that the user doesn't have to get the order correct.

`dg scaffold defs inline-component path/to/defs --typename MyType`